package.json

```
@@ -23,6 +23,8 @@
     "@androz2091/discord-invites-tracker": "1.1.0",
     "@pixelfactory/privatebin": "2.6.1",
     "@scnetwork/api": "1.0.1",
+    "@twurple/api": "^5.1.6",
+    "@twurple/auth": "^5.1.6",
```

```suggestion
    "@twurple/api": "5.1.6",
    "@twurple/auth": "5.1.6",
```

please pin dependencies, otherwise the renovatebot will scream at us - also it is [recommended](https://docs.renovatebot.com/dependency-pinning/) for node.js apps by Mend.